integration-tests/smoke/keeper_test.go

```diff
@@ -120,11 +99,11 @@ func TestKeeperBasicSmoke(t *testing.T) {
 			)
 			gom := gomega.NewGomegaWithT(t)
 
-			actions.CreateKeeperJobs(t, chainlinkNodes, registry, contracts.OCRv2Config{})
+			actions.CreateKeeperJobsLocal(t, chainlinkNodes, registry, contracts.OCRv2Config{})
```

Comment:
@tateexon Please, also, remove `*testing.T` from the helpers. You can use `errors.Wrap(err, ...)` to make it more descriptive.

Reply:
Updated in commit https://github.com/smartcontractkit/chainlink/pull/10155/commits/dc51b12bbab6bee6f8637cab61cc0f7d6b8e4dc6